views/index.handlebars

```diff
@@ -21,93 +20,66 @@
 
 <body>
 
-  <nav class="orange darken-1">
+    <nav class="orange darken-1">
     <div class="nav-wrapper">
       <a href="#" class="brand-logo">Cornucopia</a>
       <ul id="nav-mobile" class="right hide-on-med-and-down">
-        <li>
-          <div class="input-field col s6">
-            <input placeholder="Search for a recipe..." id="recipe" type="text" class="validate">
-
-          </div>
-        </li>
+        <li><div class="input-field col s6">
+          <input placeholder="Search for a recipe..." id="recipe" type="text" class="validate">
+        </div></li>
         <li><a href="#" id="search"><i class="fas fa-search"></a></i>
         {{#unless user}}
           <li><a href="#">My Pantry</a></li>
           <li><a id="login-modal-button" href="#">Sign Up/Log in</a></li>
         {{/unless}}
         {{#if user}}
-          <li><a id="pantry-modal-button" href="#">My Pantry</a></li>
+          <li><a href="#">My Pantry</a></li>
```

Comment:
This a tag needs to have an id of pantry-modal-button

`<li><a id="pantry-modal-button" href="#">My Pantry</a></li>`

Reply:
what am I doing wrong? I never touched this portion of the code.